apim/3.x/README.adoc

```diff
@@ -1,33 +1,33 @@
 [[graviteeio-api-management-helm-chart]]
+[[graviteeio-api-management-helm-chart]]
 == Gravitee.io API Management Helm Chart
 
-*Chart supported versions: 3.0.x and higher*
+Installing APIM on Kubernetes is easy with the help of our Helm Chart. **This Helm Chart supports versions: 3.0.x and higher** and will deploy the following: 
+
+* Gravitee Management API.
+* Gravitee Management UI.
+* Gravitee Portal UI.
+* Gravitee Gateway.
+* MongoDB replica set or PostgresSQL (optional dependency).
+* Elasticsearch Cluster (optional dependency).
+
+=== Prerequisites
+
+. https://helm.sh/docs/intro/install/[Helm v3].
+. MongoDB or PostgreSQL. (Not to worry, this Helm Chart will deploy these.)
+. Elasticsearch. (This Helm Chart will also deploy this.)
 
-=== Components
 
-This chart will deploy the following:
 
-* Gravitee Management API
-* Gravitee Management UI
-* Gravitee Portal UI
-* Gravitee Gateway
-* MongoDB replica-set (optional dependency)
-* Elasticsearch cluster (optional dependency)
+=== Install The Helm Chart Repo
 
-=== Installing
+Add the Gravitee.io Helm charts repo using the commmand below: 
 
-* Add the Gravitee.io helm charts repo
-+
 ....
 $ helm repo add graviteeio https://helm.gravitee.io
 ....
-* Install it
-+
-....
-$ helm install --name graviteeio-apim3x graviteeio/apim3
-....
 
-=== Create a chart archive
+=== Create A Chart Archive
```

Comment:
I think creating this archive is not a part of the installation process.
Anyway, this command will do nothing good cause it uses the current user directory ; and we don't indicate where to launch it.
Maybe it's just confusing and we can remove this "Create A Chart Archive" section ?